nethack_server/src/clientcmd.c

```diff
@@ -613,6 +622,40 @@ ccmd_get_topten(json_t * params)
     client_msg("get_topten", jmsg);
 }
 
+static void
+ccmd_get_board_entries(json_t * params)
+{
+    struct nh_board_entry *entries;
+    int listlen, i;
+    char *since;
+    json_t *jmsg, *jarr, *jobj;
+    if (json_unpack(params, "{ss}", "get_entries_since", &since) == -1)
+        exit_client("Bad parameters for get_board_entries");
+
+    entries = db_get_board_entries(since, &listlen);
+
+    jarr = json_array();
+    for (i = 0; i < listlen; i++) {
+        jobj =
+            json_pack
+            ("{si,si,si,si,si,si,si,si,si,si,si,si,si,si,ss,ss,ss,ss,ss,ss,ss}",
```

Comment:
the coding style for everything else has generally been to put the opening paren of a method call next to the method name, no whitespace in between